frontend/src/components/SavedGamesList.tsx

```diff
@@ -0,0 +1,23 @@
+import {useEffect, useState} from "react";
+import {getUserById} from "../services/UserService";
+import {SavedGame} from "../models/SavedGame";
+import SavedGameCard from "./SavedGameCard";
+import {User} from "../models/User";
+
+export default function SavedGamesList() {
+
+    const [games, setGames] = useState<SavedGame[]>([])
+
+    useEffect(() => {
+        getUserById("TEST_ID")
+            .then((user: User) => setGames(user.savedGames))
+            .catch(console.error)
+    }, [])
```

Comment:
If you only need the games from a specific/loggedin user you maybe want to create an enpoint for that?

Reply:
I will consider that 👍